graphql/database/backgroundImages/getRandomBackgroundImage.js

```diff
@@ -1,17 +1,26 @@
 import { sample } from 'lodash/collection'
 import BackgroundImageModel from './BackgroundImageModel'
-
+import { BACKGROUND_IMAGE_LEGACY_CATEGORY } from '../constants'
 /**
  * Fetch a Random image from the image base.
  * @return {Promise<BackgroundImage>}  A promise that resolve
  * into a BackgroundImage instance.
  */
 
-export default async userContext => {
-  try {
-    const images = await BackgroundImageModel.getAll(userContext)
-    return sample(images)
-  } catch (e) {
-    throw e
+export default async (
+  userContext,
+  category = BACKGROUND_IMAGE_LEGACY_CATEGORY
+) => {
+  let images
+  if (category === BACKGROUND_IMAGE_LEGACY_CATEGORY) {
+    images = await BackgroundImageModel.getAll(userContext)
```

Comment:
Add comment: We're assuming legacy photo items in the DB don't have a category. Can remove this if we migrate that field to all items.